### PR TITLE
Some text edits.

### PR DESCRIPTION
**Pull request by dblodgett-usgs**
Related to #154

### CODE REVIEW

File: data/siteText.yaml [hunk 1]
```diff
@@ -2,26 +2,26 @@ title: "### How much water do we use? U.S. water withdrawals, 1950-present"
...
-  where we use water for the last 65 years. Below, total state water use, 
-  and water use by category, is represented by the size (area) of each state. 
-  National water use for each year and by category are shown as bars above the timeline. 
-  Drag the timeline to see data for available years, and mouse over states and bars above 
-  timeline for water use values.
+  where we use water for the last 65 years. State water use is shown by the size (area) 
+  of each state. National water use for years it was estimated is shown as bars above the 
+  timeline. Drag the timeline to see data for available years, and mouse over states and bars 
+  to see water use values. The values shown are in million gallons per day (mgd). As a reference, 
+  a million gallons of water would cover a football field about 2.8 feet deep.
```
Comment (jordansread) [inline data/siteText.yaml:9]: we might want to move this reference part to below the fig?

File: data/siteText.yaml [hunk 2]
```diff
@@ -2,26 +2,26 @@ title: "### How much water do we use? U.S. water withdrawals, 1950-present"
...
 
 explanationText: |
   How and where Americans use water has changed with our population growth and many technological, 
   environmental, and regulatory drivers. In this figure, the area and color of each state polygon 
-  show the amount of water withdrawn in that state, either in total (blue) or by use category. 
-  Mouse over the legend to see maps for specific water use categories, and slide the time bar left 
-  or right to see different years.
+  show the amount of water withdrawn in that state, either in total for the four categories shown
+  (blue) or broken out by use category. Mouse over the legend to see maps for specific water use
```
Comment (jordansread) [inline data/siteText.yaml]: "click the legend"? the mouseover just highlights the part of the legend.

File: data/siteText.yaml [hunk 3]
```diff
@@ -2,26 +2,26 @@ title: "### How much water do we use? U.S. water withdrawals, 1950-present"
...
+  categories, and slide the time bar left or right to see different years.
 
-  Every 5 years since 1950, the USGS compiles and estimates water-use information in cooperation with State, 
-  Federal, and local agencies to document how the Nation's water resources are used. Water use data has been 
-  collected and compiled by the USGS since 1950 in five year increments. This information is essential to accurately 
-  understand how future water demands will be met while maintaining adequate water quality and quantities for human 
-  and ecosystem needs. This figure shows state-level data, but data at the site, county, and major watershed levels 
-  are also being collected wherever possible. The USGS Water Availability and Use Program hopes to move toward even 
-  higher spatial resolution and more frequent reports in the future.
+  Every 5 years since 1950, the USGS has compiled and estimated water-use information in cooperation with State,
```
Comment (jordansread) [inline data/siteText.yaml]: compiled what? Should be "compiled estimates" or "compiled x and estimated ..."

File: data/siteText.yaml [hunk 4]
```diff
@@ -2,26 +2,26 @@ title: "### How much water do we use? U.S. water withdrawals, 1950-present"
...
-  Every 5 years since 1950, the USGS compiles and estimates water-use information in cooperation with State, 
-  Federal, and local agencies to document how the Nation's water resources are used. Water use data has been 
-  collected and compiled by the USGS since 1950 in five year increments. This information is essential to accurately 
-  understand how future water demands will be met while maintaining adequate water quality and quantities for human 
-  and ecosystem needs. This figure shows state-level data, but data at the site, county, and major watershed levels 
-  are also being collected wherever possible. The USGS Water Availability and Use Program hopes to move toward even 
-  higher spatial resolution and more frequent reports in the future.
+  Every 5 years since 1950, the USGS has compiled and estimated water-use information in cooperation with State, 
+  Federal, and local agencies to document how the Nation's water resources are used in the year preceding the 
+  comilation. This information is essential to accurately understand how future water demands will be met while
```
Comment (jordansread) [inline data/siteText.yaml]: compilation?

File: data/siteText.yaml [hunk 5]
```diff
@@ -2,26 +2,26 @@ title: "### How much water do we use? U.S. water withdrawals, 1950-present"
...
-  collected and compiled by the USGS since 1950 in five year increments. This information is essential to accurately 
-  understand how future water demands will be met while maintaining adequate water quality and quantities for human 
-  and ecosystem needs. This figure shows state-level data, but data at the site, county, and major watershed levels 
-  are also being collected wherever possible. The USGS Water Availability and Use Program hopes to move toward even 
-  higher spatial resolution and more frequent reports in the future.
+  Every 5 years since 1950, the USGS has compiled and estimated water-use information in cooperation with State, 
+  Federal, and local agencies to document how the Nation's water resources are used in the year preceding the 
+  comilation. This information is essential to accurately understand how future water demands will be met while 
+  maintaining adequate water quality and quantities for human and ecosystem needs. This figure shows state-level 
+  data, but data at the site, county, and major watershed levels have also been collected variously over the years.
```
Comment (jordansread) [inline data/siteText.yaml]: re-word "variously"

**Review by jordansread — COMMENTED** [review]
couple of minor edits suggested

File: data/siteText.yaml (hunk 1 above)
Comment (dblodgett-usgs) [reply data/siteText.yaml:9]: Yeah... or to a stand alone call out. Pretty important that we beat people over the head with units.

File: data/siteText.yaml (hunk 3 above)
Comment (dblodgett-usgs) [reply data/siteText.yaml]: This refers to compiling sources and producing a complete estimate of consistent categories. I've got some modified text coming.

File: data/siteText.yaml (hunk 1 above)
Comment (ldecicco-USGS) [reply data/siteText.yaml:9]: mind if I merge this? Then I can fiddle with moving the text around.

Comment (jordansread) [reply data/siteText.yaml:9]: whoops, thought that was blodgett asking for a merge.

**Pull request merged**